package/src/bin/omicron-package.rs

```diff
@@ -359,23 +382,58 @@ async fn get_package(
         }
         PackageSource::Local { .. } | PackageSource::Composite { .. } => {
             progress.set_message("bundle package".into());
-            package
-                .create_with_progress_for_target(&progress, &target, package_name, &output_directory)
-                .await
-                .with_context(|| {
-                    let msg = format!("failed to create {package_name} in {output_directory:?}");
-                    if let Some(hint) = &package.setup_hint {
-                        format!("{msg}\nHint: {hint}")
-                    } else {
-                        msg
-                    }
-                })?;
+
+            let output_file = match package.output {
+                PackageOutput::Zone { .. } => {
+                    output_directory.join(format!("{}.tar.gz", package_name))
+                }
+                PackageOutput::Tarball => {
+                    package.get_output_path(package_name, output_directory)
+                }
+            };
```

Comment:
Because this is already implemented as:

https://github.com/oxidecomputer/omicron-package/blob/7caf0ba7c2b36e3308b0efe52df8d94c04bf38fc/src/package.rs#L328-L347

I think we could do the following:

```suggestion
            let output_file = package.get_output_path(package_name, output_directory);
```

Reply:
Oh awesome! I got here in a ... roundabout way, but glad to see this is done in a better way.